From 02e047e360663e81fe47a332727ec9fd7fa29474 Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Thu, 19 Dec 2024 03:08:32 +0000
Subject: [PATCH 2/2] Fix No module named 'six.moves error on ubuntu 24.04 with
 python3.12.

Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>
---
 tools/grit/grit/util.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/grit/grit/util.py b/tools/grit/grit/util.py
index 6e8cdb0eb..8398ad8a1 100644
--- a/tools/grit/grit/util.py
+++ b/tools/grit/grit/util.py
@@ -18,7 +18,6 @@ from xml.sax import saxutils
 
 import six
 from six import StringIO
-from six.moves import html_entities as entities
 
 from grit import lazy_re
 
-- 
2.43.0

