Upstream-Status: Pending

diff -wur git.original/src/mm/dumpmemory.hpp git/src/mm/dumpmemory.hpp
--- git.original/src/mm/dumpmemory.hpp	2024-09-24 11:25:39.769462195 +1200
+++ git/src/mm/dumpmemory.hpp	2024-09-24 11:26:26.429863119 +1200
@@ -197,6 +197,11 @@
 	}
 };
 
+template<class T>
+bool operator==(const DumpMemoryAllocator <T>&, const DumpMemoryAllocator <T>&) { return true; }
+ 
+template<class T>
+bool operator!=(const DumpMemoryAllocator <T>&, const DumpMemoryAllocator <T>&) { return false; }
 
 /**
  * Dump memory allocation, used for for ENABLE_MEMCHECK.
