From 5026aa3eb118853bda69bdd0a42f416b177fae7b Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@cn.fujitsu.com>
Date: Fri, 31 May 2019 16:54:21 +0900
Subject: [PATCH] Added aarch64_be into support CPU.

Signed-off-by: Lei Maohui <leimaohui@cn.fujitsu.com>
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 95d6642..cde3b5d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -42,6 +42,7 @@ AC_SUBST(HOST_OS)
 
 case "${host_cpu}" in
     arm*|sa110)		HOST_CPU="arm" ;;
+    aarch*)             HOST_CPU="aarch64" ;;
     cris*)		HOST_CPU="cris" ;;
     mips*)		HOST_CPU="mips" ;;
     powerpc|powerpc64|powerpc64le)	HOST_CPU="ppc" ;;
-- 
2.17.1

