From 69fa594af5a09e356319cc0ec2e71809ac964d9b Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Wed, 18 May 2022 10:28:53 +0900
Subject: [PATCH] Makefile: Fix build error of cross build.

Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d2f143b..127ba90 100644
--- a/Makefile
+++ b/Makefile
@@ -69,7 +69,7 @@ user: iscsiuio/Makefile
 	@echo "Read README file for detailed information."
 
 iscsiuio/Makefile: iscsiuio/configure iscsiuio/Makefile.in
-	cd iscsiuio; ./configure $(WITHOUT_ARG) --sbindir=$(SBINDIR)
+	cd iscsiuio; ./configure $(WITHOUT_ARG) --sbindir=$(SBINDIR) $(CONFIGURE_ARGS)
 
 iscsiuio/configure: iscsiuio/configure.ac iscsiuio/Makefile.am
 	cd iscsiuio; autoreconf --install
-- 
2.25.1

