From 280fbda492256d2ef70c6d377e8bd8935e7a4f9c Mon Sep 17 00:00:00 2001
From: Lei Maohui <leimaohui@fujitsu.com>
Date: Fri, 20 May 2022 17:59:03 +0900
Subject: [PATCH] Modified path of systemd service from /usr/lib to /lib.

Signed-off-by: Lei Maohui <leimaohui@fujitsu.com>
---
 etc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/Makefile b/etc/Makefile
index 2d98027..d24fd03 100644
--- a/etc/Makefile
+++ b/etc/Makefile
@@ -7,7 +7,7 @@ ifeq ($(TOPDIR),)
 	TOPDIR = ..
 endif
 
-prefix = /usr
+prefix = 
 DESTDIR ?=
 SBINDIR ?= /sbin
 
-- 
2.25.1

