diff -Naur meta-java.orig/recipes-core/cacao/cacao_git.bb meta-java/recipes-core/cacao/cacao_git.bb
--- meta-java.orig/recipes-core/cacao/cacao_git.bb	2025-07-10 02:49:55.090319951 +0000
+++ meta-java/recipes-core/cacao/cacao_git.bb	2025-07-03 00:19:02.327848264 +0000
@@ -15,7 +15,6 @@
            file://0001-java.in-Do-not-use-hardcode-paths.patch \
            file://0002-cacao-use-system-s-boehm-garbage-collector.patch \
 "
-S = "${WORKDIR}/git"
 
 inherit java autotools-brokensep update-alternatives pkgconfig features_check
 
diff -Naur meta-java.orig/recipes-core/classpathx/gnujaf_1.1.1.bb meta-java/recipes-core/classpathx/gnujaf_1.1.1.bb
--- meta-java.orig/recipes-core/classpathx/gnujaf_1.1.1.bb	2025-07-10 02:49:55.091319937 +0000
+++ meta-java/recipes-core/classpathx/gnujaf_1.1.1.bb	2025-07-03 01:30:48.905335122 +0000
@@ -12,7 +12,7 @@
 
 inherit java-library
 
-S = "${WORKDIR}/activation-${PV}"
+S = "${UNPACKDIR}/activation-${PV}"
 
 export JAVAC = "javac"
 
diff -Naur meta-java.orig/recipes-core/classpathx/gnumail_1.1.2.bb meta-java/recipes-core/classpathx/gnumail_1.1.2.bb
--- meta-java.orig/recipes-core/classpathx/gnumail_1.1.2.bb	2025-07-10 02:49:55.092319923 +0000
+++ meta-java/recipes-core/classpathx/gnumail_1.1.2.bb	2025-07-03 02:24:20.371047988 +0000
@@ -11,7 +11,7 @@
 
 inherit java-library autotools
 
-S = "${WORKDIR}/mail-${PV}"
+S = "${UNPACKDIR}/mail-${PV}"
 
 DEPENDS += "gnujaf inetlib"
 
diff -Naur meta-java.orig/recipes-core/jacl/jacl_1.4.1.bb meta-java/recipes-core/jacl/jacl_1.4.1.bb
--- meta-java.orig/recipes-core/jacl/jacl_1.4.1.bb	2025-07-10 02:49:55.092319923 +0000
+++ meta-java/recipes-core/jacl/jacl_1.4.1.bb	2025-07-03 23:57:52.944731557 +0000
@@ -13,7 +13,7 @@
 
 inherit java-library
 
-S = "${WORKDIR}/jacl${PV}"
+S = "${UNPACKDIR}/jacl${PV}"
 
 do_compile() {
   mkdir -p build
diff -Naur meta-java.orig/recipes-core/jakarta-commons/commons-net_1.4.1.bb meta-java/recipes-core/jakarta-commons/commons-net_1.4.1.bb
--- meta-java.orig/recipes-core/jakarta-commons/commons-net_1.4.1.bb	2025-07-10 02:49:55.093319908 +0000
+++ meta-java/recipes-core/jakarta-commons/commons-net_1.4.1.bb	2025-07-03 07:47:04.108589607 +0000
@@ -7,7 +7,7 @@
 
 SRC_URI = "http://archive.apache.org/dist/commons/net/source/${BP}-src.tar.gz"
 
-S = "${WORKDIR}/${BP}"
+S = "${UNPACKDIR}/${BP}"
 
 DEPENDS += "oro"
 
diff -Naur meta-java.orig/recipes-core/jakarta-commons/jakarta-commons.inc meta-java/recipes-core/jakarta-commons/jakarta-commons.inc
--- meta-java.orig/recipes-core/jakarta-commons/jakarta-commons.inc	2025-07-10 02:49:55.093319908 +0000
+++ meta-java/recipes-core/jakarta-commons/jakarta-commons.inc	2025-07-03 05:08:28.956520506 +0000
@@ -7,7 +7,7 @@
 
 inherit java-library
 
-S = "${WORKDIR}/${BP}-src"
+S = "${UNPACKDIR}/${BP}-src"
 
 # Directory in which the projects sources are located.
 MAINSOURCES = "src/java"
diff -Naur meta-java.orig/recipes-core/jdepend/jdepend_2.9.1.bb meta-java/recipes-core/jdepend/jdepend_2.9.1.bb
--- meta-java.orig/recipes-core/jdepend/jdepend_2.9.1.bb	2025-07-10 02:49:55.094319894 +0000
+++ meta-java/recipes-core/jdepend/jdepend_2.9.1.bb	2025-07-04 00:09:06.839250503 +0000
@@ -8,7 +8,6 @@
 
 SRC_URI = "git://github.com/clarkware/jdepend;protocol=https;branch=master"
 SRCREV = "57980590313a5dbde236a3eb2c8958e9e53e6a10"
-S = "${WORKDIR}/git"
 
 inherit java-library
 
diff -Naur meta-java.orig/recipes-core/servlet-api/servlet2.3_4.1.37.bb meta-java/recipes-core/servlet-api/servlet2.3_4.1.37.bb
--- meta-java.orig/recipes-core/servlet-api/servlet2.3_4.1.37.bb	2025-07-10 02:49:55.095319880 +0000
+++ meta-java/recipes-core/servlet-api/servlet2.3_4.1.37.bb	2025-07-03 03:02:34.736845810 +0000
@@ -8,7 +8,7 @@
 
 inherit java-library
 
-S = "${WORKDIR}/apache-tomcat-${PV}-src/servletapi"
+S = "${UNPACKDIR}/apache-tomcat-${PV}-src/servletapi"
 
 DEPENDS = "fastjar-native"
 
diff -Naur meta-java.orig/recipes-core/xml-commons/xml-commons-resolver1.1_1.2.bb meta-java/recipes-core/xml-commons/xml-commons-resolver1.1_1.2.bb
--- meta-java.orig/recipes-core/xml-commons/xml-commons-resolver1.1_1.2.bb	2025-07-10 02:49:55.095319880 +0000
+++ meta-java/recipes-core/xml-commons/xml-commons-resolver1.1_1.2.bb	2025-07-03 01:44:05.788893966 +0000
@@ -8,7 +8,7 @@
 
 inherit java-library
 
-S = "${WORKDIR}/xml-commons-resolver-${PV}"
+S = "${UNPACKDIR}/xml-commons-resolver-${PV}"
 
 DEPENDS = "fastjar-native jaxp1.3"
 
