diff -Naur meta-virtualization.orig/recipes-containers/lxc/lxc_git.bb meta-virtualization/recipes-containers/lxc/lxc_git.bb
--- meta-virtualization.orig/recipes-containers/lxc/lxc_git.bb	2025-07-10 02:49:14.756498126 +0000
+++ meta-virtualization/recipes-containers/lxc/lxc_git.bb	2025-07-03 07:11:33.822541551 +0000
@@ -52,8 +52,6 @@
 SRCREV = "fe31d844e882d5cc176a7935a93b14b4b2823992"
 PV = "v6.0.3"
 
-S = "${WORKDIR}/git"
-
 # Let's not configure for the host distro.
 #
 PTEST_CONF = "${@bb.utils.contains('DISTRO_FEATURES', 'ptest', '-Dtests=true', '', d)}"
diff -Naur meta-virtualization.orig/recipes-extended/irqbalance/irqbalance_git.bb meta-virtualization/recipes-extended/irqbalance/irqbalance_git.bb
--- meta-virtualization.orig/recipes-extended/irqbalance/irqbalance_git.bb	2025-07-10 02:49:14.757498122 +0000
+++ meta-virtualization/recipes-extended/irqbalance/irqbalance_git.bb	2025-07-04 05:04:07.657367107 +0000
@@ -13,6 +13,5 @@
            file://0001-add-void-to-fix-strict-prototypes.patch \
           "
 
-S = "${WORKDIR}/git"
 
 CFLAGS += "-Wno-error=format-security"
diff -Naur meta-virtualization.orig/recipes-networking/openvswitch/openvswitch_git.bb meta-virtualization/recipes-networking/openvswitch/openvswitch_git.bb
--- meta-virtualization.orig/recipes-networking/openvswitch/openvswitch_git.bb	2025-07-10 02:49:14.757498122 +0000
+++ meta-virtualization/recipes-networking/openvswitch/openvswitch_git.bb	2025-07-09 00:39:05.063491126 +0000
@@ -13,7 +13,6 @@
 	python3-resource findutils which diffutils \
 	"
 
-S = "${WORKDIR}/git"
 PV = "3.5.0"
 CVE_VERSION = "3.5.0"
 
