diff -Naur meta-cloud-services.orig/meta-openstack/recipes-extended/novnc/novnc_git.bb meta-cloud-services/meta-openstack/recipes-extended/novnc/novnc_git.bb
--- meta-cloud-services.orig/meta-openstack/recipes-extended/novnc/novnc_git.bb	2025-07-10 02:49:28.742452067 +0000
+++ meta-cloud-services/meta-openstack/recipes-extended/novnc/novnc_git.bb	2025-07-04 05:21:20.222443370 +0000
@@ -9,8 +9,6 @@
 
 SRC_URI = "git://github.com/kanaka/noVNC.git;branch=master;protocol=https"
 
-S = "${WORKDIR}/git"
-
 do_compile() {
     :
 }
