diff -Naur meta-java-orig/conf/layer.conf meta-java/conf/layer.conf
--- meta-java-orig/conf/layer.conf	2025-04-18 02:23:23.759981556 +0000
+++ meta-java/conf/layer.conf	2025-04-18 02:24:03.555084952 +0000
@@ -7,7 +7,7 @@
 BBFILE_COLLECTIONS += "meta-java"
 BBFILE_PATTERN_meta-java := "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-java = "10"
-LAYERSERIES_COMPAT_meta-java = "styhead"
+LAYERSERIES_COMPAT_meta-java = "styhead walnascar whinlatter"
 LAYERDEPENDS_meta-java += "openembedded-layer"
 
 LICENSE_PATH += "${LAYERDIR}/licenses"
