diff -Naur meta-xilinx-orig/meta-xilinx-core/conf/layer.conf meta-xilinx/meta-xilinx-core/conf/layer.conf
--- meta-xilinx-orig/meta-xilinx-core/conf/layer.conf	2024-11-19 01:48:07.634715295 +0000
+++ meta-xilinx/meta-xilinx-core/conf/layer.conf	2024-11-19 01:56:58.534232402 +0000
@@ -25,7 +25,7 @@
 LAYERDEPENDS_xilinx = "core"
 LAYERRECOMMENDS_xilinx = "openembedded-layer"
 
-LAYERSERIES_COMPAT_xilinx = "scarthgap"
+LAYERSERIES_COMPAT_xilinx = "styhead walnascar"
 
 SIGGEN_EXCLUDE_SAFE_RECIPE_DEPS += " \
   *->xrt \
