diff -Naur meta-java-orig/conf/layer.conf meta-java/conf/layer.conf
--- meta-java-orig/conf/layer.conf	2024-11-19 01:56:12.457887601 +0000
+++ meta-java/conf/layer.conf	2024-11-19 01:57:35.514496937 +0000
@@ -7,7 +7,7 @@
 BBFILE_COLLECTIONS += "meta-java"
 BBFILE_PATTERN_meta-java := "^${LAYERDIR}/"
 BBFILE_PRIORITY_meta-java = "10"
-LAYERSERIES_COMPAT_meta-java = "scarthgap"
+LAYERSERIES_COMPAT_meta-java = "styhead walnascar"
 LAYERDEPENDS_meta-java += "openembedded-layer"
 
 LICENSE_PATH += "${LAYERDIR}/licenses"
