diff -Naur meta-erlang-orig/conf/layer.conf meta-erlang/conf/layer.conf
--- meta-erlang-orig/conf/layer.conf	2025-07-11 07:59:46.026121250 +0000
+++ meta-erlang/conf/layer.conf	2025-06-30 06:37:13.234744607 +0000
@@ -7,7 +7,7 @@
 BBFILE_COLLECTIONS += "erlang-layer"
 BBFILE_PATTERN_erlang-layer := "^${LAYERDIR}/"
 BBFILE_PRIORITY_erlang-layer = "7"
-LAYERSERIES_COMPAT_erlang-layer = "walnascar styhead"
+LAYERSERIES_COMPAT_erlang-layer = "walnascar styhead whinlatter"
 
 LAYERDEPENDS_erlang-layer = "core"
 
